code/obj/flock/cage.dm

```diff
@@ -281,10 +282,18 @@
 		if(src.health <= 0)
 			qdel(src)
 
+	mob_resist_inside(var/mob/user)
+		if (ON_COOLDOWN(src, "resist_damage", 3 SECONDS))
+			return
+		ON_COOLDOWN(src, "move_damage", 1 SECOND)
+		user.show_text("<span class='alert'>[src] [pick("begins to splinter","cracks open slightly","becomes a little less solid","loosens around you")].</span>")
+		src.takeDamage("brute",6)
+		user.playsound_local(src, "sound/misc/flockmind/flockdrone_grump[pick(1,2,3)].ogg", 50, 1, 0, 0.5 )
```

Comment:
Not sure about the Flockdrone noises, not sure if cages making the same noises Flockdrones do makes sense

Reply:
I dunno, aloe suggested it. They're pitched down so they do at least sound distinct.

Reply:
Also, isn't the sound going to play over the already existing sound for flipping inside?

Reply:
yep, but they're very different sounds so it just sounds like one is in reaction to the other